packages/client/src/components/Typography.tsx

```diff
@@ -32,6 +32,18 @@ export const Paragraph = styled.p<ParagraphProps>`
   font-weight: 400;
   text-align: ${({ center }) => (center ? 'center' : 'left')};
 `
+export const ParagraphSmall = styled.p`
+  margin: 0;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 400;
+`
 export const BoldParagraph = styled(Paragraph)`
   font-weight: 700;
 `
+
+export const PlaneButton = styled.button`
```

Comment:
Maybe we should create another file where we can put smaller shared components? Button does not belong in typography